scripts/globals/npc_util.lua

```diff
@@ -349,8 +349,8 @@ function npcUtil.completeQuest(player, area, quest, params)
     end
 
     -- key item(s), fame, gil, bayld, xp, and title
-    if params["keyItem"] ~= nil then
-        npcUtil.giveKeyItem(player, params["keyItem"])
+    if params["ki"] ~= nil then
+        npcUtil.giveKeyItem(player, params["ki"])
```

Comment:
For anyone else passing-by, this is the important change here!

Looks good, thanks! 👍

Reply:
I'd really like to see the terms ki and keyItem be interchangeable everywhere. we have a high likely hood of future users using the "wrong" one no matter which we keep, so I'd rather we plan for it than simply swap out. I had thought we already did this for the actual global table, just not for the params we're using in the utils functions 

If the "right" one is nil and the other not, lets just populate/duplicate it over before we do the rest of execution.

Reply:
Yeah, agree with this

Reply:
@TeoTwawki 
Just for this case:
https://github.com/project-topaz/topaz/commit/f828092c6ff49a030f331626bf3ac82e915603e8